pulsar-broker-common/src/main/java/org/apache/pulsar/broker/web/RateLimitingFilter.java

```diff
@@ -50,9 +71,11 @@ public void init(FilterConfig filterConfig) throws ServletException {
     public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
             throws IOException, ServletException {
         if (limiter.tryAcquire()) {
+            rateLimitRequestCounter.add(1, Result.ACCEPTED.attributes);
```

Comment:
can we overload this add function to be `rateLimitRequestCounter.add(Result.ACCEPTED.attributes);` only?

Reply:
Unfortunately these counter classes are final in OpenTelemetry, so adding or modifying their functionality is not possible without some serious hacking.